Solutions/Pure Storage/SolutionMetadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "publisherId": "purestorageinc1707205060945",
```

Comment:
Hi @skeerthivasan,
In your partner center we have seen `purestoragemarketplaceadmin` publisherid. Please replace this and repackage again. Thanks